backend/geonature/migrations/versions/02e9b8758709_update_notifications_import_done.py

```diff
@@ -0,0 +1,57 @@
+"""update notifications import-done to make them generic with respect to the destination
+
+Revision ID: 02e9b8758709
+Revises: 439dd64e9cd3
+Create Date: 2024-01-23 16:10:58.149517
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '02e9b8758709'
+down_revision = '439dd64e9cd3'
+branch_labels = None
+depends_on = ("485a659efdcd",) # [import] add 'IMPORT-DONE' notification
```

Comment:
```suggestion
depends_on = None
```
Inutile car la révision précédente de la branche GeoNature a déjà une dépendance pour la dernière version d’import, qui inclus la notification import-done.